src/ynEIGEN/EigenStrategyManager.sol

```diff
@@ -306,7 +347,16 @@ contract EigenStrategyManager is
                     asset,
                     strategies[asset].userUnderlyingView((address(node)))
                 );
-                stakedBalances[j] += strategyBalance;
+
+                uint256 strategyWithdrawalQueueBalance;
+                uint256 queuedShares = node.queuedShares(strategies[asset]);
```

Comment:
we can do a bundled read for:

queuedShares and withdrawnAsset that returns a tuple so we don't do 2 separate calls to `node`

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/164/commits/2ae7917ab19770f4420e5081fe0a1b496a7fc7e3